cmd/cli/install.go

```diff
@@ -201,16 +203,19 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 }
 
 func (i *installCmd) run(config *helm.Configuration) error {
+	fmt.Print("Validating command options")
 	if err := i.validateOptions(); err != nil {
 		return err
 	}
 
+	log.Trace().Msgf("run(): start resolveValues()")
 	// values represents the overrides for the OSM chart's values.yaml file
 	values, err := i.resolveValues()
 	if err != nil {
 		return err
 	}
 
+	fmt.Print("Start installation using Helm charts")
```

Comment:
This print doesn't seem to be useful either because Helm is the only supported way of installing OSM using `osm install`. I don't see the need to print additional stuff here at the moment. Could you pull this out into a separate PR if needed?